argo/base/service-account.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: argo
+  namespace: kubeflow
```

Comment:
@asdfsx, the metadata/namespace field is inherited from the kustomization.go code in kfctl. When the manifests for argo are created during kfctl's `build` stage, the namespace field is added to metadata.